src/StorageManager/path_tree.py

```diff
@@ -52,7 +52,8 @@ def _walk(self, root):
         try:
             pathnames = os.listdir(root)
         except Exception as e:
-            self.logger.warning("Ignore path %s due to exception %s" % (root, e))
+            self.logger.warning("Ignore path %s due to exception %s" %
```

Comment:
please use `self.logger.warning("xxx %s %d", yyy, zzz)`, do not format the logging string first before calling warning, one of reason people use logger is when logger is disabled, this style will have less performance impact than yours.

Same below.